### DISCUSSION TITLE
Location background theme does not affect cards colour

### DISCUSSION THREAD

**Discussion by elydpg** [breezy-weather/breezy-weather#1113] [Ideas]
### Steps to reproduce

1. Choose a location. Make sure "day/night mode for locations" is enabled
2. Change its theme to be "always day" if it is currently night, and "always night" if it is currently day

### Expected behavior

The cards change to match the theme

### Actual behavior

The cards do not change. See the demo below:

https://github.com/breezy-weather/breezy-weather/assets/8622264/f8babdca-e2d5-4cc1-8804-d0affd96e35a

### Weather source used

N/A

### Breezy Weather version

5.2.4 standard

### Android version

12

### Device

Samsung Galaxy s10e

### Other details

This might be a case of poor documentation and not a bug. If the setting "day/night mode for locations" is enabled, then the location cards colour always matches the day/night time for the location, regardless of the background theme. If it is disabled, then the cards colour follows the system theme, again regardless of the background theme. If this is the correct intended behaviour, then the option to edit a location's theme should probably say explicitly "Background Theme".

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
It has never been implemented that way. I'm not sure what we should do.
- I'm not sure I want to add another condition of top of all the tricky logic about theming. It's already not easy to test every existing cases.
- The wording matches the one from the wallpaper settings screen

Will leave to Needs review and decide later

**Reply by elydpg** [reply]
> The wording matches the one from the wallpaper settings screen

Do you mean this wallpaper settings screen?

![Screenshot_20240610-225453_Breezy Weather.png](https://github.com/breezy-weather/breezy-weather/assets/8622264/40b6622c-9a5c-4f10-a81b-61689af9d739)

If so, then I'm referring strictly to the word "theme" in this screen, which does not appear on that screen:

![20240610_225723.png](https://github.com/breezy-weather/breezy-weather/assets/8622264/3d3bef7d-a84c-45ac-b6ba-351ed1ac9c7e)

This should probably be changed to "Background Theme" or perhaps simply "Background" or "Wallpaper", to distinguish it from the colour of the cards

**Comment by papjul** [comment]
Closing as the feature no longer exists since v6.0.3-alpha
